Windows: Fixes bad structure member in callbacks

dgmcdona opened the pull request:
This fixes a bug in the x64 callbacks symbols. The `NotificationRoutine`
is currently an `unsigned int` instead of a void pointer. This prevents
the correct mapping of the notification routine to the kernel module
that contains it.

Also bumps the version number for the netstat plugin's `modules`
requirement - it didn't get updated after #1173 was merged.

dgmcdona committed:
Windows: Fixes bad structure member in callbacks

This fixes a bug in the x64 callbacks symbols. The `NotificationRoutine`
is currently an `unsigned int` instead of a void pointer. This prevents
the correct mapping of the notification routine to the kernel module
that contains it.

atcuno commented:
@ikelos this was found in some large scale testing. Verified as fixed with the patch.

dgmcdona committed:
Windows: Bumps netstat module version requirement

This is a bump of the version number for the netstat plugin's `modules`
requirement - it didn't get updated after #1173 was merged.

ikelos reviewed (approved):
Ok, if you're happy I'm happy
